massgen/orchestrator.py

```
@@ -3212,6 +3243,39 @@ def _check_restart_pending(self, agent_id: str) -> bool:
+                planning_tool_name = tool_name
+                break
+
+        if planning_tool_name:
+            try:
+                logger.info(f"[Orchestrator] Clearing task plan for {agent_id} via {planning_tool_name}")
+                result, _ = await agent.backend._execute_mcp_function_with_retry(
+                    planning_tool_name,
+                    "{}",  # No arguments needed
+                )
+                logger.info(f"[Orchestrator] Clear task plan result for {agent_id}: {result}")
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)